lib/codecs/src/encoding/format/syslog.rs

```diff
@@ -486,24 +534,26 @@ mod tests {
         }
     }
 
-    fn serialize_to_syslog(rfc: SyslogRFC, preamble: &str) {
+    fn serialize_to_syslog(rfc: SyslogRFC, add_log_source: bool, add_event_data: bool) -> String {
         let _tz_scope = TZScope::new();
         let config = SyslogSerializerConfig{
+            add_log_source: add_log_source,
```

Comment:
where does this get used?

Reply:
These functions here are for tests. This specific function is used in the above `add_log_source` tests.